remote/remote_test.go

```diff
@@ -47,6 +54,20 @@ func TestRemote(t *testing.T) {
 	readonlyDockerRegistry.Start(t)
 	defer readonlyDockerRegistry.Stop(t)
 
+	customDockerConfigDir, err := ioutil.TempDir("", "test.docker.config.custom.dir")
+	h.AssertNil(t, err)
+	defer os.RemoveAll(customDockerConfigDir)
+
+	var customPrivileges = make(map[string]h.ImagePrivileges)
```

Comment:
Maybe you can define all of the consts in a map, and then loop on that map and initialize the `customPrivileges`.
In this way, if we'll decide to add/remove images, we'll need to do it only in one place.